scripts/jobs/cron_tasks.php

```diff
@@ -84,6 +92,16 @@
 			$webserver->writeConfigs();
 			$webserver->createOwnVhostStarter();
 			$webserver->reload();
+
+			if(1 && $run_nrp) {
```

Comment:
unecessary condition if (1) is always true...

```suggestion
			if($run_nrp) {
```